amgut/db/patches/0042.sql

```diff
@@ -0,0 +1,64 @@
+-- create the new many-to-many relation between barcodes and surveys
+CREATE TABLE ag.source_barcodes_surveys (
+	barcode              varchar  ,
+	survey_id            varchar
```

Comment:
I think this column should also be a FK. You are only making barcode a FK, but this one too. Also, both should be NOT NULL.